### ISSUE TITLE
Mixing yarn and npm causes warnings

### ISSUE THREAD

**Issue by mattgodbolt** [mattgodbolt/compiler-explorer#1343]
I've noticed nowadays I often get:

```
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```

And I wonder if we're using both `npm` and `yarn`, and if so, which one we should use. Maybe this is a side-effect of using `npm` get get `yarn` ?

**Comment by RabsRincon** [comment]
We now exclusively use npm
